app/Repositories/Summit/DoctrineSpeakerRepository.php

```diff
@@ -905,6 +913,7 @@ public function getAllByPage(PagingInfo $paging_info, Filter $filter = null, Ord
 SELECT COUNT(DISTINCT(ID)) AS QTY
 FROM (
 	SELECT S.ID,
+    M.ExternalUserID AS MemberUserExternalID,
```

Comment:
please align spaces it and add MemberID to count query

Reply:
done